Cargo.toml

```diff
@@ -18,10 +18,13 @@ nb = "1.0.0"
 fugit = "0.3.6"
 fugit-timer = "0.1.3"
 heapless = "0.7.16"
-bbqueue = { version = "0.5.0", optional=true }
+bbqueue = { version = "0.5.0", optional = true }
 
 [dev-dependencies]
+env_logger = "0.6"
+log = "0.4"
 mockall = "0.11.2"
+serialport = { git = "https://github.com/dbrgn/serialport-rs", branch = "embedded-hal", features = ["embedded"], default_features = false }
```

Comment:
I'm not sure if we should add dependencies (if only Dev) just for examples?
Is this the usual way, or should we just give a hint, if necessary?

Reply:
Yes, that's the usual way, dev dependencies are used for tests and examples 🙂 It will only be pulled in when working on the crate itself, not when depending on it.

The fact that this isn't a released dependency is not ideal, but since it's targeting e-h alpha, I thought it wouldn't make sense to merge it into the released version of serialport-rs (see https://github.com/serialport/serialport-rs/pull/59). Once e-h 1.0 is released, the changes can be integrated into upstream. For a dev dependency, I feel that this is OK.

Reply:
All right, makes sense. Thanks for clarification!